app/models/grda_warehouse/hud/client.rb

```diff
@@ -1428,6 +1428,9 @@ def hmis_client_response
     end
 
     def email
+      # Fetch the data from the source clients if we are a destination client
+      return source_clients.map(&:email).reject(&:blank?).first if destination?
```

Comment:
Is this only needed because of OP HMIS? Are the `hmis_client_response`s present on the destination client? Because if they're only on destination client, then they would be missed with this change.

Reply:
Looks like these are joined to source clients (always, as far as I can tell).  I'm unclear how it was working before.